Questie.lua

```diff
@@ -23,7 +23,7 @@ function Questie:OnInitialize()
     Questie.db = LibStub("AceDB-3.0"):New("QuestieConfig", QuestieOptionsDefaults:Load(), true)
 
     -- Initialize essential tracker tables immediately to prevent race conditions with quest events
-    QuestieTracker.InitializeTables()
+    -- QuestieTracker.InitializeTables() -- This function doesn't exist, commenting out to fix initialization
```

Comment:
I'm pretty sure it's suppose to be 

```suggestion
    QuestieTracker.Initialize()
```

Reply:
neither of these fixes will work, will just create a chain reaction of new issues.

Reply:
<img width="3286" height="1664" alt="image" src="https://github.com/user-attachments/assets/16fb1058-faa5-420e-8f8f-e95c79ea0dc3" />

As you can see, I have line 26 commented out and I am in the game with questie fully loaded and have been using it for a couple hours now without issue. I am not trying to be argumentative, but I am curious as to why you think it is causing issues or not working for you?